Python/tldnPTPumpCount.py

```diff
@@ -122,12 +122,12 @@ def OnNotify(self,state,id,events):
         if not PtWasLocallyNotified(self.key):
             return
         else:
-            if type(actTrigger.value) == type([]) and len(actTrigger.value) > 0:
+            if isinstance(actTrigger.value, list) and len(actTrigger.value) > 0:
```

Comment:
The engine will provide us with the correct type attribute value.
`if actTrigger.value:`